feature-libs/organization/administration/components/shared/detail/disable-info/disable-info.component.ts

```diff
@@ -51,7 +51,7 @@ export class DisableInfoComponent<T extends BaseItem> implements OnInit {
       disabledCreate: false,
       disabledEdit: true,
       disabledEnable: true,
-      disabledDisable: true,
```

Comment:
Should use a feature level condition to set this to avoid breaking change.

Should suffice:
```
disabledDisable: this.featureLevelService.isLevel('3.2')
```

Reply:
1 functional changes are allowed without breaking changes support
2 its fix for not released feature yet
@marlass Please confirm

Reply:
>2 its fix for not released feature yet

This makes it not a breaking change. Don't need feature flag here